site/en/docs/web-platform/region-capture/index.md

```diff
@@ -96,6 +93,10 @@ const cropTarget = await CropTarget.fromElement(mainContentArea);
 
 Unlike an `Element`, a `CropTarget` object is [serializable]. It can be passed to another document using [`Window.postMessage()`], for instance.
 
+{% Aside %}
+Prior to Chrome&nbsp;106, `CropTarget.fromElement(mainContentArea)` would reject if the type for `mainContentArea` was not `<div>` or iframe.
```

Comment:
Shall we match `<div>` with `<iframe>`?

Reply:
I've used "iframe" for consistency in the article.